components/viz/common/quads/draw_quad_perftest.cc

```diff
@@ -71,8 +71,13 @@ class DrawQuadPerfTest : public testing::Test {
   }
 
   void CleanUpRenderPass() {
+#if BUILDFLAG(IS_COBALT)
```

Comment:
Add a short note about why we do this (it's impossible to tell just by reading the code)